src/view/editor/parts/editor-meeting-rooms.test.tsx

```diff
@@ -126,29 +71,33 @@ describe('editor meeting rooms', () => {
 				type: 'Location'
 			};
 		});
-		setupFilledAppStatusStore(items);
+		const handler = getCustomResources(items);
```

Comment:
IMO the _handler_ name is misleading. The returned value actually is a "soap" response object and not a function that _handle_ the request